dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -256,7 +256,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 
                 <script>
                     window.guardian = ${windowGuardian};
-                    window.guardian.queue = []; // Queue for functions to be fired by polyfill.io callback
+                    window.guardian.onPolyfilledCallbacks = []; // Queue for functions to be fired by polyfill.io callback
```

Comment:
not sure this is meant to be in here?

```suggestion
                    window.guardian. queue = []; // Queue for functions to be fired by polyfill.io callback
```